src/components/Header/Header.tsx

```diff
@@ -29,6 +31,7 @@ const Header = ({
             {locationCity}
           </p>
         )}
+        <p className="header__date">{date}</p>
```

Comment:
As this is optional you have to check to see if it has been given before displaying it. 

On the Calendar page, you display an empty P tag as the date prop has not been given you do not want this.

Add a check similar to the checks below before rendering.